sources/packages/backend/apps/api/src/route-controllers/application/_tests_/application.students.controller.getApplicationProgressDetails.e2e-spec.ts

```diff
@@ -290,6 +296,73 @@ describe("ApplicationStudentsController(e2e)-getApplicationProgressDetails", ()
           firstCOEStatus: COEStatus.completed,
           assessmentTriggerType: application.currentAssessment.triggerType,
           hasBlockFundingFeedbackError: true,
+          hasEcertFailedValidations: false,
+        });
+    },
+  );
+
+  it(
+    "Should get application progress details with ecert failed validations array having invalid SIN for " +
+      "an application with its first disbursement having pending schedule status when " +
+      "the student's inValidSIN is false and the offering intensity is part time.",
+    async () => {
+      // Arrange
+      const application = await saveFakeApplicationDisbursements(
+        db.dataSource,
+        { student },
+        {
+          applicationStatus: ApplicationStatus.Completed,
+          offeringIntensity: OfferingIntensity.partTime,
+          firstDisbursementInitialValues: {
+            coeStatus: COEStatus.completed,
+            disbursementScheduleStatus: DisbursementScheduleStatus.Pending,
+          },
+        },
+      );
+      const [firstDisbursement] =
+        application.currentAssessment.disbursementSchedules;
+
+      const msfaaNumber = createFakeMSFAANumber(
+        {
+          student,
+          referenceApplication: application,
+        },
+        {
+          msfaaState: MSFAAStates.Signed,
+        },
+      );
+      await db.msfaaNumber.save(msfaaNumber);
+
+      firstDisbursement.msfaaNumber = msfaaNumber;
+      await db.disbursementSchedule.save(firstDisbursement);
```

Comment:
The MSFAA can be created before the `saveFakeApplicationDisbursements` and added as a property of the `firstDisbursementInitialValues`.